org.lflang.tests/src/org/lflang/tests/TestBase.java

```diff
@@ -447,11 +448,11 @@ private void validate(LFTest test, IGeneratorContext context) {
                     test.result = Result.VALIDATE_FAIL;
                 }
             }
-        } catch (Exception e) {
+        } catch (Exception | AssertionError e) {
```

Comment:
I don't think this works properly. We expect the exception to be thrown by line 442 here (`validator.validate`). If that happens, then the test log will remain empty because the exception is ignored here

I think this catch block should throw a TestExecutionException with the `e` exception here as a cause, and the printing be handled in the parent block